### ISSUE TITLE
Can't Add Kanji Card to Anki

### ISSUE THREAD

**Issue by SecretArmy** [FooSoft/yomichan#1070]
some error pop up on the chrome console:
```
yomichan.js:187 Yomichan v20.11.1.0 has encountered a problem.
Originating URL: chrome-extension://ogmnaimimemjmbakcfefmnahgdfhfami/fg/float.html?query=%E5%B6%BD&type=kanji&wildcards=off
TypeError: Cannot read property 'length' of undefined
    at DisplayFloat._getDefinitionPrimaryExpressionAndReading (chrome-extension://ogmnaimimemjmbakcfefmnahgdfhfami/mixed/js/display.js:1430:46)
    at DisplayFloat._createNote (chrome-extension://ogmnaimimemjmbakcfefmnahgdfhfami/mixed/js/display.js:1389:48)
    at DisplayFloat._addDefinition (chrome-extension://ogmnaimimemjmbakcfefmnahgdfhfami/mixed/js/display.js:1350:33)
    at async DisplayFloat._noteAdd (chrome-extension://ogmnaimimemjmbakcfefmnahgdfhfami/mixed/js/display.js:1066:28)

Issues can be reported at https://github.com/FooSoft/yomichan/issues 
```

Chrome Version 87.0.4280.66 (Official Build) (64-bit)
Yomichan ver 20.11.1.0
Win 10 Pro 64-bit.

**Comment by toasted-nutbread** [comment]
Duplicate of #1068; fixed by #1069. Queued for [20.11.1.1 patch release](https://github.com/FooSoft/yomichan/issues/980#issuecomment-733785189), whenever FooSoft has time.

**Comment by SecretArmy** [comment]
Oh, already fixed. Then will close this now. Thanks.
